sql/core/src/test/resources/sql-tests/inputs/postgreSQL/create_view.sql

```diff
@@ -0,0 +1,778 @@
+-- Portions Copyright (c) 1996-2019, PostgreSQL Global Development Group
+--
+-- CREATE VIEW
+-- https://github.com/postgres/postgres/blob/REL_12_STABLE/src/test/regress/sql/create_view.sql
+
+-- Spark doesn't support geometric types
```

Comment:
Although we will not do that, shall we have a JIRA issue and ID here?

Reply:
sure, ok.